Fabric.Authorization.Client/Fabric.Authorization.Client.csproj

```diff
@@ -2,14 +2,43 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard1.6</TargetFramework>
+    <Authors>Fabric.Authorization.Client</Authors>
+    <Company>Health Catalyst</Company>
+    <Version>1.0.0.0-local</Version>

```

Comment:
does the normal build process also update this minor/sub versions?

Reply:
Yes, this will be changed with some command line magic.  I copied these values from our other projects, but they will be transformed